app/utils/review.js

```diff
@@ -0,0 +1,76 @@
+import { Alert, Linking } from 'react-native';
+import RNUserDefaults from 'rn-user-defaults';
```

Comment:
We shouldn't be using RNUserDefaults here as it is shared between both apps.
This way the user would maybe review experimental instead official.
If we user AsyncStorage here, we're going to ask them to review twice (official and experimental) and can be annoying, but it's probably the best solution.